src/pybind/mgr/dashboard/frontend/src/styles/defaults/_bootstrap-defaults.scss

```diff
@@ -17,62 +15,62 @@ $black: #000 !default;
 $blue: #007bff !default;
 $indigo: #6610f2 !default;
 $purple: #6f42c1 !default;
-$pink: #a94442;
-$red: #ef5c55;
+$pink: #a94442 !default;
+$red: #ef5c55 !default;
 $orange: #fd7e14 !default;
-$yellow: #ffc200;
-$green: #0b0;
+$yellow: #ffc200 !default;
+$green: #0b0 !default;
 $teal: #20c997 !default;
 $cyan: #17a2b8 !default;
 
-$primary: #2b99a8;
-$secondary: #374249;
-$success: $green;
-$info: $primary;
-$warning: $yellow;
-$danger: $red;
-$light: $gray-100;
-$dark: #777;
+$primary: #2b99a8 !default;
+$secondary: #374249 !default;
+$success: $green !default;
+$info: $primary !default;
+$warning: $yellow !default;
+$danger: $red !default;
+$light: $gray-100 !default;
+$dark: #777 !default;
 
-$fg-color-over-dark-bg: $white;
-$fg-hover-color-over-dark-bg: $gray-500;
+$fg-color-over-dark-bg: $white !default;
+$fg-hover-color-over-dark-bg: $gray-500 !default;
 
 $theme-colors: (
-  'accent': #ef5c55,
+  'accent': $red,
```

Comment:
I'd recommend not to use the variable here, but the actual hash value. In future it might happen, that a vendor wants to change the red color tone but the accent color isn't always the red color.

Otherwise, please add a variable $accent and assign this instead.

Reply:
@Exotelis As per our previous conversation, let's go for the $accent variable approach.